### ISSUE TITLE
missing checksums at test/get_previous_releases.py

### ISSUE THREAD

**Issue by katesalazar** [bitcoin/bitcoin#23098]
It can download v22.0, v0.21.x but it won't unpack them automatically
because checksums are missing from
https://github.com/bitcoin/bitcoin/blob/16ccb3a1cd9125eb24a5b45a98099ff98660767a/test/get_previous_releases.py#L23-L62

**Expected behavior**

It downloads the requested old tag binaries and unpacks them automatically.

**Actual behavior**

It downloads the requested old tag binaries and then it is showing an output:
```
Checksum for given version doesn't exist
```

So the developer has to read the source code of the program to find out
next steps.

**To reproduce**

Try:

```
/PATH/TO/BITCOIN/test/get_previous_releases.py -b v22.0 v0.21.2
```

You might want to remove the version's binary and directory before
trying, or I think you can remove the whole old releases directory
using the -r argument to get_previous_releases.

**System information**

N/A

**Other**

You might want to update this lines accordingly as well:

https://github.com/bitcoin/bitcoin/blob/c427a5800bb53208d30eeb03a73ab8be879e5f45/test/functional/feature_backwards_compatibility.py#L34-L47

And at:

https://github.com/bitcoin/bitcoin/blob/c427a5800bb53208d30eeb03a73ab8be879e5f45/test/README.md#L101

---

If you fix this, thank you very much!

**Comment by katesalazar** [comment]
If the fix is adding those checksums, remember you might want to back port
0.21 checksums to the v22 branch.

**Comment by MarcoFalke** [comment]
None of the tests use the 22.0 binary yet, so I don't see a reason to add it

**Comment by katesalazar** [comment]
> None of the tests use the 22.0 binary yet, so I don't see a reason to add it

I supposed this could really be a non issue,
but thought opening a ticket gives a good balance
between expressiveness, low noise, and async attention.

You could still ponder adding a note explaining why 22 (or 22 and 21)
weren't included in the demo command at test/README.

**Comment by MarcoFalke** [comment]
If you have a suggestion on how to improve the documentation, you can leave a comment at #19013 or create a new pull request with the improvement.

**Comment by katesalazar** [comment]
> If you have a suggestion on how to improve the documentation, you can leave a comment at 19013 or create a new pull request with the improvement.

(I removed the linking feature from your text because it's already linked
and linking again right away would be noisy).

I could submit a little PR adding a small note on why the line looks dated,
and calling to be (at least partially) superseded by PR 19013,
but fighting the future is a lost cause. The v0.20.2 tag is almost here!
Upon release my comment would become obsolete.

It would make more sense to me instead, to point out a possible 
release process modification, in which pondering a modification
of these lines would be encouraged as a step before tagging:
https://github.com/bitcoin/bitcoin/blob/c427a5800bb53208d30eeb03a73ab8be879e5f45/test/README.md#L97-L106

There you could add simply "Releases vX.Y.Z... didn't add enough
behavioral difference to be recommended as standalones in
get_previous_releases.py" before line "By default, up to 4 tests
will be run in parallel by...".

For that reason I'm going to CC @laanwj and please excuse I don't
remember who are the five or six new release managers.

**Comment by fanquake** [comment]
Closing this for now. Any discussion can continue in #19013.
